src/main/java/com/capitalone/dashboard/webhook/sonarqube/SonarQubeHookController.java

```diff
@@ -33,4 +36,9 @@ public SonarQubeHookController(SonarQubeHookService sonarQubeHookService) {
                 .status(HttpStatus.CREATED)
                 .body(response);
     }
+
```

Comment:
for the validation to be done on the request object please add`@Validated` under `@RestController`